modules/gapi/src/backends/streaming/gstreamingbackend.cpp

```diff
@@ -0,0 +1,211 @@
+//
+// Copyright 2020 Intel Corporation.
+//
+// This software and the related documents are Intel copyrighted materials,
+// and your use of them is governed by the express license under which they
+// were provided to you ("License"). Unless the License provides otherwise,
+// you may not use, modify, copy, publish, distribute, disclose or transmit
+// this software or the related documents without Intel's prior written
+// permission.
+//
+// This software and the related documents are provided as is, with no
+// express or implied warranties, other than those that are expressly
+// stated in the License.
+//
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/gapi/util/throw.hpp> // throw_error
+#include <opencv2/gapi/streaming/streaming.hpp> // kernels
+
+#include "api/gbackend_priv.hpp"
+#include "backends/common/gbackend.hpp"
+
+#include "gstreamingbackend.hpp"
+#include "gstreamingkernel.hpp"
+
+namespace {
+
+struct StreamingCreateFunction
+{
+    static const char *name() { return "StreamingCreateFunction";  }
+    cv::gapi::streaming::CreateActorFunction createActorFunction;
+};
+
+using StreamingGraph = ade::TypedGraph
+    <
+      cv::gimpl::Op
+    , StreamingCreateFunction
+    >;
+
+using ConstStreamingGraph = ade::ConstTypedGraph
+    <
+      cv::gimpl::Op
+    , StreamingCreateFunction
+    >;
+
+
+class GStreamingExecutable final: public cv::gimpl::GIslandExecutable
+{
+    virtual void run(std::vector<InObj>  &&,
+                     std::vector<OutObj> &&) override {
+        // NB: Is it really needed in non-streaming  mode ?
+        GAPI_Assert(false && "Not implemented");
+    }
+
+    virtual void run(GIslandExecutable::IInput &in,
+                     GIslandExecutable::IOutput &out) override;
+
+    virtual bool canReshape() const override { return false;  }
+    virtual void reshape(ade::Graph&, const cv::GCompileArgs&) override {
+        cv::util::throw_error(std::logic_error("GStreamingExecutable::reshape() is not supported"));
```

Comment:
The same question

Reply:
Your backend must support reshape.

Reply:
...since it costs nothing to support it.

Reply:
Implemented in the most ugliest way (the only way I know :(). Added a test - seems to be working just fine (please, check that `Reshape` test actually tests reshape of the backend).

Reply:
Will have a look, thanks